paper/paper.md

```diff
@@ -0,0 +1,140 @@
+
+3. [cdp-frontend](https://github.com/CouncilDataProject/cdp-frontend): A TypeScript library containing all frontend components and a compiled web application. The web application allows for simple data exploration and sharing. It additionally acts as a method for quickly understanding context of events, voting records, and more because of it allows users to interactively explore the data produced by the backend pipelines. We separate the web application from the instance so that we can easily release new features, visualizations, an UX improvements whenever we release a new version of `cdp-frontend`.
+
+## Cookiecutter and the Produced Repository
+
+`cookiecutter-cdp-deployment` will generate all necessary files for an entirely new CDP instance as well as additional setup documentation for the user to follow to fully complete the instance deployment process.
+
+Utilizing [GitHub Actions](https://github.com/features/actions) and [GitHub Pages](https://pages.github.com/), data processing and web hosting are entirely free as long as the user sets their instance's GitHub repository visibility to public. While the limited primary costs of CDP tools are:
+
+1. [Google Speech-to-Text](https://cloud.google.com/speech-to-text/) for transcript generation.
+2. [Firebase Cloud Firestore](https://firebase.google.com/docs/firestore/) for event metadata storage and access.
+3. [Firebase Storage](https://firebase.google.com/docs/storage) for file storage and access.
+
+![CDP Core Infrastructure Figure. Event Gather Pipeline: inputs, GitHub Action, Python Pipeline tasks, and storage. Event Index Pipeline: inputs, GitHub Action, Python n-gram generation, and storage. Web Application Building: inputs, GitHub Action, and GitHub Pages.\label{fig:core-infra}](./assets/cdp_core_infrastructure.png)
+
+CDP tools allow for decentralized control over the management and deployment of each CDP instance while producing a standardized open-access dataset for both research and for municipal transparency and accessibility.
+
+## Data Access
+
+Once data is processed by a CDP instance, it is available through that instance's interactive web application.
+
+![CDP Web Application. Screenshot of a single event's page. Navigation tabs for basic event details such as the minutes items, the entire transcript, and voting information. Additionally both the transcript search and the full transcript have links to jump to a specific sentence in the meeting.](./assets/event-page-screenshot.png)
```

Comment:
Add caption to this image to bring in line with previous image

Reply:
As in, this description isn't good or I'm just missing the fig text?

Reply:
Fig text has been added was just missing the `\label`

Reply:
The former - its fixed now!